challenges-webapp/src/main/webapp/vue-app/engagement-center/components/challenges/Challenges.vue

```diff
@@ -50,6 +50,37 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           <span> {{ $t('challenges.filter.searchTooltip') }} </span>
         </v-tooltip>
       </div>
+      <div class="filter_menu pt-1">
+        <v-menu
+          v-model="showFilterMenu"
+          offset-y>
+          <template #activator="{ on }">
+            <button
+              class="btn"
+              v-on="on"
+              @blur="closeFilterMenu">
+              <span class="text-truncate d-none d-lg-inline">
+                {{ filterChallengeLabel }}
+              </span>
+              <v-icon class="uiIconMiniArrowDown uiIconLightGray filter_menuIcon" small />
+            </button>
+          </template>
+          <v-list>
+            <v-list-item @mousedown="$event.preventDefault()">
+              <v-list-item-title @click="filter ='ALL'">{{ $t('challenges.filter.allChallenges') }}</v-list-item-title>
```

Comment:
define `@click` on parent element for better UX.
BTW, I think we are using a different UX from Stream Filter combo box (It's better to harmonize and use a simple `select` element here).

Reply:
done!